payload/Setup/update_homebrew.bash

```
@@ -86,22 +85,27 @@
 export ENABLE_CLANG_FORMAT=on
 # need to make this dynamic
-#export CMAKE_OSX_DEPLOYMENT_TARGET=14.2
+export CMAKE_OSX_DEPLOYMENT_TARGET=15.1
 export CC=clang
 export CXX=clang
-export CPP=clang
-export CMAKE_PROGRAM_PATH=${PATH}:/Applications/Xcode.app/Contents/Developer/usr/bin
-#export SSL_CERT_DIR=??
+export CPP=clang-cpp
+export CMAKE_PROGRAM_PATH=${PATH}:${HOMEBREW_PREFIX}/opt/llvm@19/bin:${HOMEBREW_PREFIX}/opt/llvm@18/bin:/Applications/Xcode.app/Contents/Developer/usr/bin
+OPENSSL_DIR=$(echo $(openssl version -a | grep -F "SSL" | grep -F "DIR" | cut -d: -f 2-2 | tr -d '"')) ;
```

Shellcheck (0.10.0)

[warning] OPENSSL_DIR appears unused. Verify use (or export if used externally).

(SC2034)